documentation.md

```diff
@@ -10,7 +10,6 @@ toc: false
 ## Pulpcore Documentation
 
 * Latest: [docs.pulpproject.org](http://docs.pulpproject.org).
```

Comment:
Would you mind changing "http" occurrences with "https"?

Reply:
We can then burn another issue down: https://github.com/pulp/pulpproject.org/issues/94.

Reply:
This issue is from year 2017. I updated links from posts from 3 years ago but did not go any further. If you're fine I will close out that issue, I do not see sense to update posts from 2011 where links are  broken or links that do not exist anymore `http://pulp-python.readthedocs.io/en/3.0.0b1/`, or  links related to pulp2

Reply:
I had in mind updating the links we currently present to our users -- like the above one where this discussion happens; not the old ones.

Reply:
That single reference was updated too.